app/server/controllers/UserController.js

```diff
@@ -706,6 +706,49 @@ UserController.checkOutById = function(id, user, callback){
   callback);
 };
 
+/**
+ * [ADMIN ONLY]
+ *
+ * Make user an admin
+ * @param  {String}   userId   User id of the user being made admin
+ * @param  {String}   user     User making this person admin
+ * @param  {Function} callback args(err, user)
+ */
+UserController.makeAdminById = function(id, user, callback){
```

Comment:
I copied the prototype from the checkIn route and controller which doesn't seem to save the user either, but expects the user argument for some reason.